applications/serial_lte_modem/src/lwm2m_carrier/slm_at_carrier.c

```diff
@@ -839,37 +835,6 @@ static int do_carrier_request_link_up(void)
 	return lwm2m_carrier_request(LWM2M_CARRIER_REQUEST_LINK_UP);
 }
 
-/* AT#XCARRIER="auto_connect","read|write"[,<auto-connect-flag>] */
-static int do_carrier_auto_connect(void)
```

Comment:
Adding @mgolu to comment to the removal of the auto_connect due to comment exchange in:
https://github.com/nrfconnect/sdk-nrf/pull/10840#issuecomment-1513959768

Reply:
Ping @mgolu.

Reply:
Sorry - Github notifications issue. 

I think moving it from run-time to configuration time is fine.